src/mono/wasm/build/WasmApp.targets

```diff
@@ -486,7 +486,23 @@
 
     <Message Text="Compiling native assets with emcc. This may take a while ..." Importance="High" />
     <Exec Command='emcc $(EmccCFlags) "%(_WasmObjectsToBuild.SourcePath)" -c -o "%(_WasmObjectsToBuild.Identity)"' EnvironmentVariables="@(EmscriptenEnvVars)" />
-    <Exec Command="emcc $(EmccLDFlags) @(_DotnetJSSrcFile->'--js-library &quot;%(Identity)&quot;', ' ') @(_BitcodeFile->'&quot;%(Identity)&quot;', ' ') @(_WasmObjects->'&quot;%(Identity)&quot;', ' ') -o &quot;$(_WasmIntermediateOutputPath)dotnet.js&quot;" EnvironmentVariables="@(EmscriptenEnvVars)" />
+
+    <ItemGroup>
+      <_EmccLinkerArguments Include="$(EmccLDFlags)" />
+      <_EmccLinkerArguments Include="--js-library %(_DotnetJSSrcFile.Identity)" />
+      <_EmccLinkerArguments Include="@(_BitcodeFile)" />
+      <_EmccLinkerArguments Include="@(_WasmObjects)" />
+      <_EmccLinkerArguments Include="-o $(_WasmIntermediateOutputPath)dotnet.js" />
```

Comment:
Sorry, I guess their format does require whitespaces to be quoted: https://github.com/emscripten-core/emscripten/blob/main/tools/response_file.py#L16-L53.  Your earlier iteration was more correct.